gpt4all-chat/chatllm.h

```diff
@@ -14,7 +14,8 @@ enum LLModelType {
     GPTJ_,
     LLAMA_,
     CHATGPT_,
-    REPLIT_
+    REPLIT_,
+    FALCON_
```

Comment:
This will require handling this value in the switch statement in ChatLLM::serialize and ChatLLM::deserialize